monocle/gerrit/review.py

```diff
@@ -116,6 +124,9 @@ def get(self, updated_since, change=None):
                     break
             else:
                 break
+        if self.prefix:
+            for review in reviews:
+                review["project"] = self.prefix + review["project"]
```

Comment:
So the repository_shortname and repository_prefix line 165 and 167 need to be fixed as we split on "/" so for prefix suc as "rdo/" it won't be as expected.